src/current/v25.4/upgrade-with-pcr.md

```diff
@@ -0,0 +1,76 @@
+
+{{site.data.alerts.callout_info}}
+The entire standby cluster must be on the same major version as the primary cluster or a major version the primary cluster [can directly upgrade to]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#compatible-versions). Within the primary and standby CockroachDB clusters, the _system virtual cluster (SystemVC)_ must be at a cluster major version greater than or equal to the _app virtual cluster (AppVC)_.
+{{site.data.alerts.end}}
+
+## Upgrade primary and standby clusters
+
+To upgrade your primary and standby clusters:
+
+1. Ensure that the virtual clusters on both your primary cluster and your standby cluster are finalized on the current version. If their versions have not been finalized, [finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#finalize-a-major-version-upgrade-manually) them before beginning the upgrade process.
+
+1. [Upgrade the binaries]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#perform-a-major-version-upgrade) on the standby cluster. Replace the binary on each node of the cluster and restart the node.
+
+1. [Finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#finalize-a-major-version-upgrade-manually) the upgrade on the standby cluster's SystemVC. 
+
+    {{site.data.alerts.callout_info}}
+    If you need to [roll back]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#roll-back-a-major-version-upgrade) an upgrade, you must do so before the upgrade has been finalized.
+    {{site.data.alerts.end}}
+
+1. [Upgrade the binaries]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#perform-a-major-version-upgrade) on the primary cluster. Replace the binary on each node of the cluster and restart the node.
+
+1. [Finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#finalize-a-major-version-upgrade-manually) the upgrade on the primary cluster's SystemVC. 
+
+    {{site.data.alerts.callout_info}}
+    If you need to [roll back]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#roll-back-a-major-version-upgrade) an upgrade, you must do so before the upgrade has been finalized. Rolling back the upgrade on the primary cluster does not roll back the standby cluster.
+    {{site.data.alerts.end}}
+
+1. [Finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#finalize-a-major-version-upgrade-manually) the upgrade on the primary cluster's AppVC. 
+
+    Upgrading the primary cluster's AppVC also upgrades the standby cluster's AppVC, since it replicates from the primary.
```

Comment:
I'm not finding any useful descriptions of how to do this, and the only search result for "AppVC" is this https://www.cockroachlabs.com/docs/stable/read-from-standby.html#how-the-read-from-standby-feature-works

Should definitely include a code snippet, and we may be missing other content on how to connect to the AppVC. Also as mentioned above, naming should be consistent and I see "application virtual cluster" more than "AppVC"